server/plugins/AntiCheatReloaded/checks.yml

```
@@ -82,7 +82,7 @@ speed:
   # Debugs as 'moved too fast on ground'
   groundSpeed:
     # Should this part of the check be enabled?
-    enabled: true
+    enabled: false
```

Don't disable ground speed :boi:
```suggestion
    enabled: true
```